test/travis-before-install.sh

```diff
@@ -19,8 +19,15 @@ travis_retry go get \
   github.com/golang/lint/golint \
   github.com/mattn/goveralls \
   github.com/modocache/gover \
-  github.com/jcjones/github-pr-status \
-  github.com/letsencrypt/goose/cmd/goose
+  github.com/jcjones/github-pr-status &
+
+(wget https://github.com/jsha/boulder-tools/raw/master/goose.gz &&
```

Comment:
I considered this, but I think it doesn't make sense. We don't protect any of the other fetches in Travis with a sha256sum, largely because the worst an attacker could do would be to produce a test success when it should have failed.